assignments/arrays.js

```diff
@@ -63,34 +63,62 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
+//console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
+console.log(`Car ${inventory[32].id} is a ${inventory[32].car_year} ${inventory[32].car_make} ${inventory[32].car_model}`);
 
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+let lastCar = `${inventory[49].car_make} ${inventory[49].car_model}`;
+console.log(lastCar);
 
```

Comment:
You can also do something like this. This will always grab the last item even if a new one gets added on
```suggestion
let last_element = my_array[my_array.length - 1];
```